### PR TITLE
update cdist/README.md

### PR DESCRIPTION
**Pull request by 4nd3r**
[Rendered](https://github.com/cdist-community/cdist/blob/4nd3r/update_readme/README.md)

Related PRs:
https://github.com/cdist-community/cdist-conf/pull/3
https://github.com/cdist-community/cdist-extra/pull/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,48 @@
...
+Work is split between three repositories:
 
-For more information have a look at [**homepage**](https://cdi.st)
-or at **``docs/src``** for manual in **reStructuredText** format.
+* [cdist](https://github.com/cdist-community/cdist) - main implementation of **cdist core**.
+* [cdist-conf](https://github.com/cdist-community/cdist-conf) - **essential** explorers and types.
+* [cdist-extra](https://github.com/cdist-community/cdist-extra) - **non-essential** explorers, types, scripts, tools etc.
 
-## Contributing
+What's the difference between essential and non-essential? Explorers and types,
```
Comment (sideeffect42) [inline README.md:13]: We should sync this up with cdist-community/cdist-conf#2 once we agreed on a definition.

File: README.md [hunk 2]
```diff
@@ -1,29 +1,48 @@
 # cdist
 
-**cdist** is a usable configuration management system.
+This is [community maintained](https://github.com/cdist-community)
+fork of Ungleich's [cdist](https://github.com/ungleich/cdist)
+(after [f061fb168ddacc894cb6e9882ff5c8ba002fadd8](https://github.com/ungleich/cdist/commit/f061fb168ddacc894cb6e9882ff5c8ba002fadd8)).
```
Comment (sideeffect42) [inline README.md:5]: Please shorten the commit ID to f061fb1.
It is too obtrusive in its full length.

File: README.md [hunk 3]
```diff
@@ -1,29 +1,46 @@
...
 
-Issues can be made and other project management activites happen
-[**only in upstream**](https://code.ungleich.ch/ungleich-public/cdist)
-(needs [**ungleich** account](https://account.ungleich.ch)).
+### Get core with essentials
 
-For community-maintained types there is
-[**cdist-contrib** project](https://code.ungleich.ch/ungleich-public/cdist-contrib).
+```bash
+git clone --recurse-submodules https://github.com/cdist-community/cdist
```
Comment (sideeffect42) [inline README.md:27]: Should we have an extra step to update the conf submodule to the latest commit?

File: README.md [hunk 4]
```diff
@@ -1,29 +1,42 @@
...
+```
+
+### Get everything
+
+```bash
+git clone https://github.com/cdist-community/cdist
+git clone https://github.com/cdist-community/cdist-conf
+git clone https://github.com/cdist-community/cdist-extra
+ln -s "$(pwd)/cdist/bin/cdist" "$HOME/.local/bin/cdist"
+cat > "$HOME/.cdist.cfg" << EOF
```
Comment (sideeffect42) [inline README.md:38]: I think this is a bit dangerous. Someone who already has cdist installed and carelessly copy-pastes these lines into their terminal would overwrite their config.

I'd at least make it `>>"${HOME:?}/.cdist.cfg`.

File: README.md (hunk 1 above)
Comment (4nd3r) [reply README.md]: Yes, initial section of README (until getting started) should be used for other 2 repos as well.

File: README.md (hunk 3 above)
Comment (4nd3r) [reply README.md:30]: Added.

File: README.md (hunk 4 above)
Comment (4nd3r) [reply README.md:53]: But we are targeting power users for now, right? :smile: I added warning in the beginning.

File: README.md (hunk 1 above)
Comment (4nd3r) [reply README.md]: Created PRs for other two repos.

File: README.md [hunk 5]
```diff
@@ -1,29 +1,56 @@
...
+ln -s "$PWD/cdist/bin/cdist" "$HOME/.local/bin/cdist"
+```
 
-Matrix and IRC are bridged.
+**NB** Your `PATH` may not contain `$HOME/.local/bin/`, so modify this line accordingly.
+
+To update `cdist/conf` submodule later:
+
+```sh
+git submodule foreach git pull origin main
```
Comment (sideeffect42) [inline README.md:40]: Do we need such a complex command?
Wouldn't `git submodule update --remote cdist/conf` do?

File: README.md [hunk 6]
```diff
@@ -1,29 +1,56 @@
...
+```
+
+### Get everything
+
+**PS** Don't copy-paste following lines carelessly into your terminal - existing `~/.cdist.cfg` will be overwritten.
```
Comment (sideeffect42) [inline README.md:45]: PS? Shouldn't this be "NB", too?

**Review by sideeffect42 — COMMENTED** [review]
Two minor comments, otherwise LGTM.

**Review by sideeffect42 — APPROVED** [review]

**Pull request merged**